### PR TITLE
W-14914426 Adding TLS feature

### PR DESCRIPTION
**Pull request by valkyrie69**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 1]
```diff
@@ -327,16 +327,47 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
 
+. In Design Center, Select *Files +* > *New file*.  
+. In the *Add new file* window, select *API format > other*, name the file `properties.json`, then click *Create*.
+. In the new file, add `"tlsEnabled": true`.
++
+Adding this property and setting it to true customizes the connector generation so that when the Rest Connect Service receives the property, the connector is upgraded to run on Mule runtime 4.6. and the `tlsEnabled` flag propagates to the connector generation. 
+
+This enables Anypoint Studio to display the appropriate form to collect the information required for TLS connection configuration. 
+ 
+It is best to add the `properties.json` file only if you want to enable TLS support, or explicitly opt out of it. If you don't add a `properties.json` file to your API project, the REST Connect Service generates the standard connector artifacts, without TLS support.
```
Comment (sbdias) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: If you don't add a properties.json  - or if you add an empty properties.json file

**Review by sbdias — COMMENTED** [review]
I have added minor suggestions

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 2]
```diff
@@ -182,20 +182,24 @@ baseUri: https://jsonplaceholder.typicode.com
...
+      (rest-connect.operationName): Retrieve a post by id
       displayName: Get a post by ID.
       responses:
         200:
           body:
             type: Post
 ----
 
-Example with REST Connect library:
+You can also change the text in `displayName`, for example:
```
Comment (liocastiglioni) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: IMHO this method is the easiest and standard, maybe it should be offered first as it was.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 3]
```diff
@@ -327,16 +327,47 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
```
Comment (liocastiglioni) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:331]: why was this part deleted? https://docs.mulesoft.com/exchange/to-deploy-using-rest-connect#oauth2-in-design-center-for-rest-connect from the original docs?. I this it was replaced by mistake and shoul stay afber "Reserved words" and "Generating Connectors with TLS support" titles

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 4]
```diff
@@ -327,16 +327,47 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Add a `properties.json` file to the root directory of your Design Center API project to enable TLS support to enhance security for your app.
```
Comment (liocastiglioni) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: I wouldn't use enhance security, Instead I would write:

Add a `properties.json` file to the root directory of your Design Center API project to enable TLS support for your app.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 5]
```diff
@@ -412,31 +443,43 @@ securitySchemes:
...
+. Log in to GitHub.
++
+When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+////
+
+== Known Limitations
 
-== Known limitations
-  - Anypoint platform, including REST Connect, does not support API fragments for OAS.
+  - Anypoint Platform, including REST Connect, does not support API fragments for OAS.
```
Comment (liocastiglioni) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: - TLS support can only be enabled for Mule 4 (SmartConnector) connectors. This feature is not supported for Mule 3 (DevKit).

**Review by liocastiglioni — CHANGES_REQUESTED** [review]
left some comments

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 1 above)
Comment (valkyrie69) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: I removed that intentionally. This is telling them to only add this file if they are enabling TLS support, so I don't see any situation where they would add an empty properties.json file.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 3 above)
Comment (valkyrie69) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc:331]: @liocastiglioni As noted in the PR, this section is outdated. The screen shots are outdated, there are missing steps, and it doesn't work as documented. If you can work with me to update it so it's current, we can do that.

Comment (liocastiglioni) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc:331]: Agree, lets remove it then

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 1 above)
Comment (liocastiglioni) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: the only situation I imagine is adding the json file and forget to add its properties. But your can leave it as is and the mail will show the error anyways

**Review by liocastiglioni — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 6]
```diff
@@ -223,13 +223,15 @@ uses:
...
 
 
+
+[[metadata-limitations]]
 == Metadata Limitations
 
 REST Connect generates metadata for each operation based on your schema definition in the request and response for each method in your RAML. REST Connect cannot generate metadata based on examples in the RAML.
 
-Because only one input/output type is supported for each operation, REST Connect selects the first one that was declared. You can change this behavior by using the 'default' property from the REST Connect library.
+REST Connect supports only one input/output type and uses the first one declared in the API specification. You can change this behavior by using the 'default' property from the REST Connect library.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
REST Connect supports only one input/output type and uses the first one declared in the API specification. You can change this behavior by using the `default` property from the REST Connect library.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 7]
```diff
@@ -223,13 +223,15 @@ uses:
...
 
-Example with REST Connect library:
+The following is an example that uses the REST Connect library:
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: Suggest: The following example uses the REST Connect library:

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 8]
```diff
@@ -262,7 +264,7 @@ uses:
 [[reserved-keywords]]
 == Reserved Keywords
 
-Valid identifiers for REST Connect must not match any of the reserved Java keywords or reserved Mule keywords.
+Valid identifiers for REST Connect must not use any reserved Java keywords or reserved Mule keywords.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
Valid identifiers for REST Connect must not use any reserved Java or Mule keywords.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 9]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate a connector, the connector is upgraded to run on Mule runtime 4.6, and TLS is enabled so that Anypoint Studio displays the appropriate form for collecting required TLS connection configuration information for the connector. 
```

Comment (dhirsch1) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: I suggest using bullets and tweaking the wording to break up the long text.

When you enable TLS support and generate a connector:

*  The connector is upgraded to run on Mule runtime 4.6. 
*  TLS is enabled, so that Anypoint Studio displays a form for collecting the connector's required TLS connection configuration information.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 10]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector. 
+
+To enable TLS, add a `properties.json` file to the root directory of your Design Center API project for your app:
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
To enable TLS, add a `properties.json` file to the root directory of the Design Center API project for your app: 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 11]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector. 
+
+To enable TLS, add a `properties.json` file to the root directory of your Design Center API project for your app: 
+
+. In Design Center, Select *Files +* > *New file*.  
+. In the *Add new file* window, select *API format > other*, name the file `properties.json`, then click *Create*.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
. In the *Add new file* window, select *API format > other*, name the file `properties.json`, and click *Create*.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 12]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+. In the new file, add `"tlsEnabled": true`.
++
+[NOTE]
+Add the `properties.json` file only if you want to enable TLS support or explicitly opt out of it. If you don't add a `properties.json` file to your API project, the REST Connect Service generates the standard connector artifacts, without TLS support.
+
+After you add the `properties.json` file and enable TLS, Anypoint Studio displays the TLS configuration form for the connector.  
+ 
+When you add a `properties.json` file to your project, ensure that the `tlsEnabled` property is correctly set with a valid boolean value of either `true` to enable TLS or `false` to explicitly disable TLS. If the `properties.json` file is invalid or if the property is incorrectly set, the connector generation is aborted.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
When you add a `properties.json` file to your project, ensure that the `tlsEnabled` property is set correctly with a valid boolean value of either `true` to enable TLS or `false` to explicitly disable TLS. If the `properties.json` file is invalid or the property is incorrectly set, the connector generation is aborted.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 13]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+
+[[upgrade-rest-connector]]
+== Upgrading a REST Connect Generated Connector
 
+To update your REST Connect generated connector to use the latest version of Mule runtime and to be compatible with Java 17:
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
To update your REST-Connect-generated connector to use the latest version of Mule runtime and to be compatible with Java 17:
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 14]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+
+. In Design Center, open your API project. 
+. Click *Publish*. 
++
+The connector is re-generated. 
+
+If you want to add <<tls-support,TLS support>> to your application, it is a good idea to add it before re-generating the connector so you only have to test your app once.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
If you want to add <<tls-support,TLS support>> to your application, add it before re-generating the connector so that you only have to test your app once. 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 15]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+
+If you want to add <<tls-support,TLS support>> to your application, it is a good idea to add it before re-generating the connector so you only have to test your app once. 
+
+//// 
+Does not work. Screen shots are outdated, steps are missing. This needs to be updated before it's published again. 
 == OAuth2 in Design Center for REST Connect
 
-. Define an API with OAuth2 - Authorization Code and one operation in Design Center. You can use the following GitHub API example:
+. Create a new API specification project named `GitHub API` in Design Center, and copy and paste the previous example into the project. 
+. Define an API with OAuth2 - Authorization Code and one operation in Design Center, for example:
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:372]: Should OAuth2 - Authorization Code be in back ticks?

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 16]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
            application/json:
              type: string
 ----
+. Click *Publish* to publish the API specification to Exchange so you can use it in your Mule app later.
+. In the Publishing to Exchange window, accept the default values and click *Publish to Exchange*.
 +
-. Create a new API specification project named Github API in Design Center, and copy and paste the example above. From the API Designer, click Publish to Exchange:
+. In Anypoint Studio, create a Mule application with an HTTP Listener, the GitHub API you created in the previous step, and a Logger.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
. In Anypoint Studio, create a Mule app with an HTTP Listener, the GitHub API you created in the previous step, and a Logger. 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 17]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
 . Because GitHub API’s base URL is api.github.com, you can put “/” in the Base Path.
-. Match and modify your external callback URL. The callback URL receives an access token from GitHub. By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it specific to your app. The demo app’s callback URL should be `+http://githubapp-smky.cloudhub.io/callback+` so I need to replace “my-app” with “githubapp-smky.” You can find this information to go to the menu and select the copy link in Design Center.
+. Match and modify your external callback URL. The callback URL receives an access token from GitHub. 
++
+By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it to be specific to your app. The demo app’s callback URL is `+http://githubapp-smky.cloudhub.io/callback+`, so replace “my-app” with “githubapp-smky.” You can find this information to go to the menu and select the copy link in Design Center.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: Use back ticks for my-app and githubapp-smky?

Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it to be specific to your app. The demo app’s callback URL is `+http://githubapp-smky.cloudhub.io/callback+`, so replace “my-app” with “githubapp-smky.” You can find this information by accessing menu and selecting the copy link in Design Center.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 18]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
 +
 image::rest-connect-ready-to-deploy.png[]
 +
 . After you get your external callback URL, specify the same URL in your GitHub settings.
 +
 image::rest-connect-auth-callback-url.png[]
 +
-. You are ready to retrieve an access token from GitHub. In this case, go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, your case would be `+http://my-app.cloudhub.io/authorize+` - replace my-app.cloudhub.io with the one you get with Copy link. When you reach this URL, your browser asks you to log into GitHub.
-. When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+You are ready to retrieve an access token from GitHub.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
You can now retrieve an access token from GitHub. 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 19]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
 . After you get your external callback URL, specify the same URL in your GitHub settings.
 +
 image::rest-connect-auth-callback-url.png[]
 +
-. You are ready to retrieve an access token from GitHub. In this case, go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, your case would be `+http://my-app.cloudhub.io/authorize+` - replace my-app.cloudhub.io with the one you get with Copy link. When you reach this URL, your browser asks you to log into GitHub.
-. When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+You are ready to retrieve an access token from GitHub. 
++
+. Go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, for example `+http://my-app.cloudhub.io/authorize+`.
+. Replace `my-app.cloudhub.io` with the one you got when you copied the link.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:480]:
```suggestion
. Replace `my-app.cloudhub.io` with the one you received when you copied the link. 
```

Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: Replace "one." Maybe use "path" instead.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 20]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
 +
-. You are ready to retrieve an access token from GitHub. In this case, go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, your case would be `+http://my-app.cloudhub.io/authorize+` - replace my-app.cloudhub.io with the one you get with Copy link. When you reach this URL, your browser asks you to log into GitHub.
-. When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+You are ready to retrieve an access token from GitHub. 
++
+. Go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, for example `+http://my-app.cloudhub.io/authorize+`.
+. Replace `my-app.cloudhub.io` with the one you got when you copied the link. 
+. Log in to GitHub.
++
+When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+`. Replace `my-app.cloudhub.io` with the path you received with `Copy link`.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 11 above)
Comment (dhirsch1) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: or "and then click *Create*."

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 21]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector. 
+
+To enable TLS, add a `properties.json` file to the root directory of your Design Center API project for your app: 
+
+. In Design Center, Select *Files +* > *New file*.  
+. In the *Add new file* window, select *API format > other*, name the file `properties.json`, then click *Create*.
+. In the new file, add `"tlsEnabled": true`.
++
+[NOTE]
+Add the `properties.json` file only if you want to enable TLS support or explicitly opt out of it. If you don't add a `properties.json` file to your API project, the REST Connect Service generates the standard connector artifacts, without TLS support.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
Add the `properties.json` file only if you want to enable TLS support or explicitly opt out of it. If you don't add a `properties.json` file to your API project, the REST Connect Service generates the standard connector artifacts without TLS support.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 13 above)
Comment (dhirsch1) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: This one is up to you. Compound adjectives are usually hyphenated, but if it seems funny to you, feel free to not use the hyphens.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 22]
```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+
+[[upgrade-rest-connector]]
+== Upgrading a REST Connect Generated Connector
 
+To update your REST Connect generated connector to use the latest version of Mule runtime and to be compatible with Java 17:
+
+. In Design Center, open your API project. 
+. Click *Publish*. 
++
+The connector is re-generated.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
The connector is regenerated. 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 23]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
+. In the Publishing to Exchange window, accept the default values and click *Publish to Exchange*.
 +
-. Create a new API specification project named Github API in Design Center, and copy and paste the example above. From the API Designer, click Publish to Exchange:
+. In Anypoint Studio, create a Mule application with an HTTP Listener, the GitHub API you created in the previous step, and a Logger. 
 +
-image::rest-connect-publish-to-exchange.png[]
+This app listens to `+https://my-app.cloudhub.io/getIssues+` and returns the results based on your search term.
 +
-. Create a simple Mule application in Design Center of an HTTP Listener, the Github API, and a Logger. This app listens to `+https://my-app.cloudhub.io/getIssues+` and returns the results based on your search term.
+image::rest-connect-dc-flow.png["Mule flow in Studio canvas that uses HTTP Listener, GitHub API, and Logger components"]
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: I don't think you need to say "in Studio canvas"?

Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
image::rest-connect-dc-flow.png["Mule flow that uses the HTTP Listener, GitHub API, and Logger components"]
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 24]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
 +
-image::rest-connect-dc-flow.png[]
+. Click GitHub API Configuration on the canvas.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: Bold GitHub API Configuration?

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 25]
```diff
@@ -412,31 +446,44 @@ securitySchemes:
...
+. Configure OAuth 2.0 with an authorization code for the connector. 
 +
-. Configure OAuth 2.0 with authorization code for a connector. Most of the fields are auto-populate based in the GitHub API specification.
+Most of the fields are auto-populated based in the GitHub API specification.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
Most of the fields are auto-populated based on the GitHub API specification.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 17 above)
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it to reflect your app. The demo app’s callback URL is `+http://githubapp-smky.cloudhub.io/callback+`, so replace “my-app” with “githubapp-smky.” You can find this information by accessing the menu and selecting the copy link in Design Center.
```

Comment (dhirsch1) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc]: What information does "this information" refer to?

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 19 above)
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:480]: the one > path ?

Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc]:
```suggestion
. Replace `my-app.cloudhub.io` with the one you received when you copied the link. 
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 20 above)
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:483]:
```suggestion
When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+`. Replace `my-app.cloudhub.io` with the path you received with `Copy link`.
```

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 11 above)
Comment (dhirsch1) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc:340]: or "and then click *Create*.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 26]
```diff
@@ -327,16 +329,53 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS).  
+
+When you enable TLS support and generate a connector:
+
+* The connector is upgraded to run on Mule runtime 4.6.
+* TLS is enabled, so that Anypoint Studio displays a form for collecting the connector's required TLS connection configuration information.
+
+To enable TLS, add a `properties.json` file to the root directory of the Design Center API project for your app: 
+
+. In Design Center, Select *Files +* > *New file*.
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:344]: Just noticed the + in *Files +*. Is that correct?

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc [hunk 27]
```diff
@@ -327,16 +329,53 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
...
+To update your REST-Connect-generated connector to use the latest version of Mule runtime and to be compatible with Java 17:
+
+. In Design Center, open your API project. 
+. Click *Publish*. 
++
+The connector is regenerated. 
+
+If you want to add <<tls-support,TLS support>> to your application, add it before re-generating the connector so that you only have to test your app once. 
+
+////
```
Comment (dhirsch1) [inline modules/ROOT/pages/to-deploy-using-rest-connect.adoc:367]: Just checking that this doesn't interfere with anything.

**Review by dhirsch1 — APPROVED** [review]

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 26 above)
Comment (valkyrie69) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc:344]: Yes, that's how it is in the UI.

File: modules/ROOT/pages/to-deploy-using-rest-connect.adoc (hunk 27 above)
Comment (valkyrie69) [reply modules/ROOT/pages/to-deploy-using-rest-connect.adoc:367]: That whole section is commented out. This won't show up. I'll double check in my local build though!
